index.html

```diff
@@ -5,7 +5,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Reusable Components</title>
     <link rel="shortcut icon" href="src/components/logo.png" type="image/x-icon" />
-    <link rel="stylesheet" type="text/css" href="dist/reusable-components.css">
```

Comment:
I haven't looked at this repo before so I'm not sure why this page exists, but why doesn't it need CSS anymore?

Reply:
never mind, found it.

Reply:
It's a demo page Adriana created.

Reply:
It's a demo/local development page; much of the point of it is so that I can just see the components in isolation in the browser.  Definitely still needs CSS.

Reply:
It does need CSS but webpack handles that for the dev build.